src/TestConsoleApp/TestConsoleApp.csproj

```
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net462</TargetFramework>
+  </PropertyGroup>
```

I know it's only a test console app, but I still think it worthwhile to add...
```suggestion
    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
  </PropertyGroup>
```